pkg/server/sandbox_status.go

```diff
@@ -61,13 +63,81 @@ func (c *criContainerdService) PodSandboxStatus(ctx context.Context, r *runtime.
 		return nil, fmt.Errorf("failed to get sandbox ip: %v", err)
 	}
 
-	info, err := sandbox.Container.Info(ctx)
+	ctrInfo, err := sandbox.Container.Info(ctx)
 	if err != nil {
 		return nil, fmt.Errorf("failed to get sandbox container info: %v", err)
 	}
-	createdAt := info.CreatedAt
+	createdAt := ctrInfo.CreatedAt
 	status := toCRISandboxStatus(sandbox.Metadata, state, createdAt, ip)
-	return &runtime.PodSandboxStatusResponse{Status: status}, nil
+	info, err := toCRISandboxContainerInfo(ctx, sandbox.Container, r.GetVerbose())
+	if err != nil {
+		return nil, fmt.Errorf("failed to get verbose sandbox container info: %v", err)
```

Comment:
Could we log the error and Include it into `info` instead of return directly?  
Even if the error is not nil,  we should return status information and indicate the cause of the error in info, like what #470 do.

Reply:
docker do it like this:
```
# docker inspect b6
[
    {
        "Id": "b6143d0814f53e50e4024ebeee321b6ac96cc840c17edd02d7f46f03708c9e34",
        "Created": "2017-12-06T03:29:37.080804775Z",
        "Path": "top",
        "Args": [],
        "State": {
            "Status": "running",
            "Running": true,
            "Paused": false,
            "Restarting": false,
            "OOMKilled": false,
            "Dead": false,
            "Pid": 34113,
            "ExitCode": 0,
            "Error": "",                 // "Error" indicates the reason of the error
            "StartedAt": "2017-12-06T03:29:37.56730597Z",
            "FinishedAt": "0001-01-01T00:00:00Z"
        },
       ... ...
```
We can include this in `info`, wdyt?

Reply:
> Could we log the error and Include it into info instead of return directly?
Even if the error is not nil, we should return status information and indicate the cause of the error in info.

Currently there is no way to include the error message, but I agree we should not fail in most cases. We should do something similar with https://github.com/kubernetes-incubator/cri-containerd/pull/475/files.

Reply:
> docker do it like this

We've included most of the information for container status. We just don't save information for sandbox container. :)

Reply:
yeah probably don't need it for the sandbox pause container... but will need to comb over the inspect for containers to make sure we're not missing anything useful.